app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Admin::UsersController < ApplicationController
+  before_action :admin_user, only: [:new, :index, :show, :edit, :update, :destroy]
```

Comment:
`%i`記法で簡潔に記載できることは知っておいてください！
https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/SymbolArray
```suggestion
  before_action :admin_user, only: %i[new index show edit update destroy]
```